tests/test__collection_api.py

```diff
@@ -3970,11 +3970,6 @@ def test__aggregate_not_implemented(self):
                 {'$project': {'a': {'$cmp': [1, 2]}}},
             ])
 
-        with self.assertRaises(NotImplementedError):
-            self.db.collection.aggregate([
-                {'$project': {'a': {'$concatArrays': [[0, 1], [2, 3]]}}},
```

Comment:
Please use another not implemented array function here, for instance `$isArray`.